.github/ISSUE_TEMPLATE/ci-flake.yaml

```diff
@@ -0,0 +1,18 @@
+name: CI Flake
+description: Report CI jobs that flaked
+labels: kind/flake
+body:
+- type: textarea
+  id: jobref
+  attributes:
+    label: Link to the job that flaked.
+  validations:
+    required: true
+
+- type: textarea
+  id: details
+  attributes:
+    label: Snippet of what failed.
+    description: Inset a short snippet of what failed to have a quick preview (stack trace) and something to track even if logs would be gone.
```

Comment:
```suggestion
    description: Insert a short snippet of what failed (stack trace) to provide a quick preview even if the logs are missing.
```

Reply:
fixed the typo but the new wording doesn't retain causality

Reply:
let's leave the wording then but the `would be gone` part is ungrammatical, imo either are or were

Reply:
ok, I can settle for `are`